matching-core/src/main/java/com/graphhopper/matching/MapMatching.java

```diff
@@ -65,10 +62,17 @@
  */
 public class MapMatching {
 
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    // Penalty in m for each U-turn performed in the path between two subsequent candidates
+    // Note that this penalty should roughly correspond to the penalty used for unfavored
+    // virtual edges because results get inconsistent otherwise.
+    private static final double VIRTUAL_NODE_U_TURN_PENALTY = 100;
```

Comment:
Can we try to make it consistent with the u-turn penalty used [here](https://github.com/graphhopper/graphhopper/blob/master/core/src/main/java/com/graphhopper/routing/weighting/TurnWeighting.java#L41) or [here](https://github.com/graphhopper/graphhopper/blob/master/core/src/main/java/com/graphhopper/util/Parameters.java#L114)?

I.e. using the same unit (seconds) and value. If both values should be similar, what happens if the heading penalty is changed?